test/hydrated_test.dart

```diff
@@ -81,8 +79,8 @@ void main() {
 
 /// An example of a class that serializes to and from a string
 class SerializedClass {
-  bool value;
-  int count;
+  late final bool value;
+  late final int count;
```

Comment:
Would it be better to not use `late` here to improve readability? It seems like there is no real need in `late` since we can use `factory` for `fromJson`. What do you think?

Reply:
Sure, makes sense!